tests/Integration/FontTest.php

```diff
@@ -40,6 +40,15 @@
 
 class FontTest extends TestCase
 {
+    public function testSetFontSpaceLimit()
+    {
+        $this->assertEquals(-50, Font::getFontSpaceLimit());
+        Font::setFontSpaceLimit(-100);
+        $this->assertEquals(-100, Font::getFontSpaceLimit());
+
+        Font::setFontSpaceLimit(-50);
```

Comment:
I couldn't run tests on a local machine, I don't have php5, but there was maybe problem with this line for next tests and I decided to add it. I can remove it and we will check tests.

Reply:
> but there was maybe problem with this line for next tests and I decided to add it.

That's a bad sign, because tests have to be independent and isolated. Either try to check `setUp` and/or `tearDown` functions or the implementation itself.

Regarding PHP5: You can change your code and commit as often as you want. Afterwards our test environment gets triggered and you will see the results below. Don't be afraid to check for edge cases etc.

Reply:
I'm trying today to get rid of this line and test on a local machine, it's not a problem! I think according to current release this fix is the best way not to break projects using this package. But I like the idea to use config. I also don't like singletons and want to have clear code, that will have predictable behavior, but in this case, maybe, this is the only way without breaking everything. Maybe configuration from config is a good idea for v.1.0